lib/containers/utils.pm

```diff
@@ -83,7 +83,7 @@ sub registry_url {
 sub runtime_smoke_tests {
     my %args = @_;
     my $runtime = $args{runtime};
-    my $image = $args{image} // registry_url('alpine', '3.6');
+    my $image = $args{image} // registry_url('alpine', 'latest');
```

Comment:
Is this intentional?

Reply:
Yes, 3.6 is not available and it's ancient. Why not using just latest?

Reply:
For consistent test results.

IMO this should be in a separate commit at least.

Reply:
done